tests/network-tests/src/fixtures/content/createContentStructure.ts

```diff
@@ -52,11 +52,12 @@ export class CreateContentStructureFixture extends BaseQueryNodeFixture {
     const contentLeaderKeyPair = this.api.getKeypair(contentLeader.role_account_id.toString())
     await this.cli.importAccount(contentLeaderKeyPair)
 
-    this.debug('Creating channel categories')
-    this.createdItems.channelCategoryIds = await this.createChannelCategories(this.channelCategoryCount)
+    // TODO: adjust tests after https://github.com/Joystream/joystream/issues/3574 has been implemented
+    // this.debug('Creating channel categories')
+    // this.createdItems.channelCategoryIds = await this.createChannelCategories(this.channelCategoryCount)
 
-    this.debug('Creating video categories')
-    this.createdItems.videoCategoryIds = await this.createVideoCategories(this.videoCategoryCount)
+    // this.debug('Creating video categories')
+    // this.createdItems.videoCategoryIds = await this.createVideoCategories(this.videoCategoryCount)
```

Comment:
Let's remove this. I will have to resolve conflicts between this branch and video category rework's branch anyway.